icees_kg/utils/jsons_to_csv.py

```diff
@@ -0,0 +1,36 @@
+"""This is a throwaway script that converts jsonl files to csvs."""
```

Comment:
I think you forgot to throw it away!

Reply:
We should probably clean up (remove?) the comment blocks.

Reply:
I removed all these extra one-off scripts. They shouldn't be needed once the pre-compute script is finalized.